test/test_all.py

```diff
@@ -703,15 +714,15 @@ class ExampleJitterNoPython(ExampleJitter):
             "'py-llvm 3.2' module is required for llvm tests"
 
         # Remove llvm tests
-        if TAGS["llvm"] not in exclude_tags:
+        if (not include_tags and TAGS["llvm"] not in exclude_tags):
```

Comment:
Maybe there is an unmanaged case where you only include a tag which will pull a dependency on tcc for example, on a machine which does not support tcc.